Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM php:7.3.2-apache
+
+RUN apt update && apt install -y \
+    unzip \
+    libzip-dev \
+    libpq-dev \
+    libldap2-dev \
+    libc-client-dev \
+    libkrb5-dev \
+    libfreetype6-dev \
+    libjpeg62-turbo-dev \
+    libpng-dev \
+  && rm -rf /var/lib/apt/lists/*
+
+RUN curl -o limesurvey.zip https://download.limesurvey.org/latest-stable-release/limesurvey3.15.9+190214.zip \
```

Comment:
Release number should not be hard-coded.

Reply:
Hi, did you have a look at this Dockerfile: https://github.com/martialblog/docker-limesurvey/blob/master/apache/Dockerfile

Reply:
Sure. But it seems hard-coded in there too?

Reply:
That is usually the case though. For example: https://github.com/nextcloud/docker/blob/master/16.0/apache/Dockerfile or https://github.com/jenkinsci/docker/blob/master/Dockerfile

Since you want some validation of what version is inside. What would you prefer?

Reply:
I'd prefer if the version would automatically be greped from the tags in the git repo.

Reply:
I like @Shnoulle 's idea of creating a separate repo for the Docker code, keeping things clean in the main repo... with the Docker examples and tests and whatnot.

You can then build the Images automatically via Travis or something like Hubot, or have one of these humans I've heard so much about do it. I created a little shell helper for that already, updating the version and checksum.

Reply:
OK

Reply:
As I said before, feel free to fork/use my repo.